### PR TITLE
Specify default logging level

### PR DESCRIPTION
**Pull request by kalyanac**

### CODE REVIEW

**Comment by nolanmar511** [comment]
Sorry -- LGTM'ed by mistake.

File: README.md [hunk 1]
```diff
@@ -105,8 +105,8 @@ export GCLOUD_PROFILER_CONFIG=./path/to/your/profiler/configuration.js
 
 The profiler writes log statements to the console log for diagnostic purposes.
 By default, the log level is set to warn. You can adjust this by setting
-`logLevel` in the config. Setting `logLevel` to 0 will disable logging, 1 sets
-log level to error, 2 sets it to warn, 3 sets it to info, and 4 sets it to
+`logLevel` in the config. Setting `logLevel` to 0 will disable logging (default),
+1 sets log level to error, 2 sets it to warn, 3 sets it to info, and 4 sets it to
```
Comment (nolanmar511) [inline README.md]: Confirmed by checking configs. 

2, not 0, is default.

Comment (nolanmar511) [reply README.md]: Noting that this needs to be addressed.

Comment (nolanmar511) [reply README.md]: Noting this should be addressed.

Comment (kalyanac) [reply README.md]: done

**Pull request merged**